daemon/daemon.go

```diff
@@ -995,14 +995,15 @@ func (daemon *Daemon) initDiscovery(config *Config) error {
 // Reload reads configuration changes and modifies the
 // daemon according to those changes.
 // These are the settings that Reload changes:
-// - Daemon labels.
-// - Daemon debug log level.
-// - Daemon insecure registries.
+// - Daemon labels
+// - Daemon debug log level
+// - Daemon insecure registries
+// - Daemon registry mirrors
```

Comment:
Non-blocking observation: I feel like none of the registry-related settings should be prefixed with "Daemon".